pkgs/development/python-modules/toolz/default.nix

```diff
@@ -6,18 +6,16 @@
 
 buildPythonPackage rec {
   pname = "toolz";
-  version = "0.10.0";
+  version = "0.11.1";
```

Comment:
```suggestion
  version = "0.11.1";
  disabled = isPy27;
```
```
  Processing ./toolz-0.11.1-py2-none-any.whl
  ERROR: Package 'toolz' requires a different Python: 2.7.18 not in '>=3.5'
cannot build derivation '/nix/store/fja1cmxfc94wxcszlfpjh5i45rsizafn-python2.7-strategies-0.2.3.drv': 1 dependencies couldn't be built
```

Reply:
otherwise LGTM

```
https://github.com/NixOS/nixpkgs/pull/101363
1 package marked as broken and skipped:
python37Packages.rl-coach

2 packages failed to build:
python27Packages.strategies python27Packages.toolz

96 packages built:
electron-cash electrum eliot-tree hwi keepkey_agent python37Packages.altair python37Packages.aplpy python37Packages.atomman python37Packages.batchgenerators python37Packages.caffe python37Packages.clifford python37Packages.cytoolz python37Packages.dask python37Packages.dask-gateway python37Packages.dask-glm python37Packages.dask-image python37Packages.dask-jobqueue python37Packages.dask-ml python37Packages.dask-mpi python37Packages.dask-xgboost python37Packages.datashader python37Packages.diofant python37Packages.distributed python37Packages.eth-utils python37Packages.glymur python37Packages.hwi python37Packages.ibis-framework python37Packages.image-match python37Packages.imagecorruptions python37Packages.imgaug python37Packages.intake python37Packages.keepkey python37Packages.keepkey_agent python37Packages.mask-rcnn python37Packages.partd python37Packages.pims python37Packages.pyfftw python37Packages.rlp python37Packages.scikitimage python37Packages.sparse python37Packages.spectral-cube python37Packages.strategies python37Packages.streamz python37Packages.stumpy python37Packages.stytra python37Packages.sunpy python37Packages.tensorly python37Packages.textacy python37Packages.toolz python37Packages.trezor python37Packages.trezor_agent python38Packages.altair python38Packages.aplpy python38Packages.atomman python38Packages.batchgenerators python38Packages.caffe python38Packages.clifford python38Packages.cytoolz python38Packages.dask python38Packages.dask-gateway python38Packages.dask-glm python38Packages.dask-image python38Packages.dask-jobqueue python38Packages.dask-ml python38Packages.dask-mpi python38Packages.dask-xgboost python38Packages.datashader python38Packages.diofant python38Packages.distributed python38Packages.eth-utils python38Packages.glymur python38Packages.image-match python38Packages.imagecorruptions python38Packages.imgaug python38Packages.intake python38Packages.keepkey python38Packages.mask-rcnn python38Packages.partd python38Packages.pims python38Packages.pyfftw python38Packages.rlp python38Packages.scikitimage python38Packages.sparse python38Packages.spectral-cube python38Packages.strategies python38Packages.streamz python38Packages.stumpy python38Packages.stytra python38Packages.sunpy python38Packages.tensorly python38Packages.textacy python38Packages.toolz python38Packages.trezor trezor_agent streamlit update-python-libraries

```

Reply:
I created a Python 2 expression instead.

Reply:
pinning of the python 2 package should probably be a separate commit from the bump. that way if we need to revert, we don't remove the python2 version as well